Common/Various/CommonUtils.swift

```diff
@@ -521,6 +521,22 @@ func formatAudioLevelChannels(channels: Int) -> String {
     return String(localized: " \(channels) ch")
 }
 
+func parseAudioLevel(_ string: String?) -> Float {
+    guard let string = string else {
+        return Float.infinity
+    }
+    if string.hasPrefix("Muted,") {
+        return Float.nan
+    } else if string.hasPrefix("Unknown,") {
+        return Float.infinity
+    } else if let range = string.range(of: " dB,") {
+        let levelString = String(string[string.startIndex ..< range.lowerBound])
+        return Float(levelString) ?? Float.nan
+    } else {
+        return Float.nan
+    }
+}
+
```

Comment:
We should add the numeric value to the remote control protocol to simplify this and make it work with bars.

Reply:
Still in progress

Reply:
Done. Although I'm not entirely satisfied because the alignment doesn't quite fit (compared to all the other elements).

What do you think? should we recreate a new audiolevenview for remotecontrol?

Reply:
We want the UI to be pretty. So I guess yes =)

Reply:
Or possibly send both what we have today and the numeric level.

Reply:
ok much cleaner now :)